modules/exploits/windows/http/manageengine_adaudit_plus_cve_2022_28219.rb

```diff
@@ -0,0 +1,365 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer
+  include Msf::Exploit::Remote::TcpServer
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::JavaDeserialization
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'ManageEngine ADAudit Plus CVE-2022-28219',
+        'Description' => %q{
+          This module exploits CVE-2022-28219, which is a pair of
+          vulnerabilities: a path traversal in the /cewolf endpoint, and a
+          blind XXE in, to upload and execute an executable file.
+        },
+        'Author' => [
+          'Ron Bowes', # Analysis and module
+          'Naveen Sunkavally', # Initial PoC + disclosure
```

Comment:
I think we normally put PoC/Disclosures first, with the module author last

```suggestion
          'Naveen Sunkavally', # Initial PoC + disclosure
          'Ron Bowes', # Analysis and module
```